substrate/frame/people/README.md

```diff
@@ -0,0 +1,93 @@
+# People Pallet
+
+A pallet managing the registry of proven individuals.
+
+## Overview
+
+The People pallet stores and manages identifiers of individuals who have proven their personhood. It
+tracks their personal IDs, organizes their cryptographic keys into rings, and allows them to use
+contextual aliases through authentication in extensions. When transactions include cryptographic
+proofs of belonging to the people set, the pallet's transaction extension verifies these proofs
+before allowing the transaction to proceed. This enables other pallets to check if actions come from
+unique persons while preserving privacy through the ring-based structure.
+
+The pallet accepts new persons after they prove their uniqueness elsewhere, stores their
+information, and supports removing persons via suspensions. While other systems (e.g., wallets)
+generate the proofs, this pallet handles the storage of all necessary data and verifies the proofs
+when used.
+
+## Key Features
+
+- **Stores Identity Data**: Tracks personal IDs and cryptographic keys of proven persons
+- **Organizes Keys**: Groups keys into rings to enable privacy-preserving proofs
+- **Verifies Proofs**: Checks personhood proofs attached to transactions
+- **Links Accounts**: Allows connecting blockchain accounts to contextual aliases
+- **Manages Registry**: Adds proven persons and will support removing them
+
+## Interface
+
+### Dispatchable Functions
+
+- `set_alias_account(origin, account)`: Link an account to a contextual alias. Once linked, this
+  allows the account to dispatch transactions as a person with the alias origin using a regular
+  signed transaction with a nonce, providing a simpler alternative to attaching full proofs.
+- `unset_alias_account(origin)`: Remove an account-alias link
+
+### Tasks
+
+- `build_ring(origin, ring_index)`: Build or update a ring's cryptographic commitment. This task
+  processes queued keys into a ring commitment that enables proof generation and verification. Since
+  ring construction, or rather adding keys to the ring, is computationally expensive, it's performed
+  periodically in batches rather than processing each key immediately. The batch size needs to be
+  reasonably large to enhance privacy by obscuring the exact timing of when individuals' keys were
+  added to the ring, making it more difficult to correlate specific persons with their keys.
```

Comment:
This file differs from lib.rs doc - onboard_people should be mentioned here.
We should also mention:
- remove_suspended_keys
- merge_queue_pages.

Reply:
Done.